eva_submission/eload_brokering.py

```diff
@@ -43,77 +28,57 @@ def __init__(self, eload_number: int, vcf_files: list = None, metadata_file: str
                 self.eload_cfg.set('validation', 'valid', 'metadata_spreadsheet', value=os.path.abspath(metadata_file))
 
     def broker(self):
+        """Run the brokering process"""
         # Reset previous values that could have been set before
         self.eload_cfg['brokering'] = {}
         output_dir = self._run_brokering_prep_workflow()
-        self._collect_brokering_worklflow_results(output_dir)
+        self._collect_brokering_workflow_results(output_dir)
```

Comment:
```suggestion
        self._collect_brokering_prep_results(output_dir)
```
because couple of lines later, there are calls to the actual brokering...